e2e-tests/functions

```diff
@@ -312,7 +317,27 @@ wait_cluster_consistency() {
 	"$(kubectl get ps "${cluster_name}" -n "${NAMESPACE}" -o jsonpath='{.status.orchestrator.ready}')" == "${orc_size}" &&
 	"$(kubectl get ps "${cluster_name}" -n "${NAMESPACE}" -o jsonpath='{.status.orchestrator.state}')" == "ready" &&
 	"$(kubectl get ps "${cluster_name}" -n "${NAMESPACE}" -o jsonpath='{.status.state}')" == "ready" ]]; do
-		echo 'waiting for cluster readyness'
+		echo 'waiting for cluster readyness (async)'
+		sleep 15
+	done
+}
+
+wait_cluster_consistency_gr() {
+	local cluster_name=${1}
+	local cluster_size=${2}
+	local router_size=${3}
+
+	if [ -z "${router_size}" ]; then
+		router_size=3
+	fi
+
+	sleep 7 # wait for two reconcile loops ;)  3 sec x 2 times + 1 sec = 7 seconds
+	until [[ "$(kubectl get ps "${cluster_name}" -n "${NAMESPACE}" -o jsonpath='{.status.mysql.state}')" == "ready" &&
+	"$(kubectl get ps "${cluster_name}" -n "${NAMESPACE}" -o jsonpath='{.status.mysql.ready}')" == "${cluster_size}" &&
+	"$(kubectl get ps "${cluster_name}" -n "${NAMESPACE}" -o jsonpath='{.status.router.ready}')" == "${router_size}" &&
```

Comment:
we can add some indentation here

Reply:
I have executed shfmt and this is what I got, so let's not change now.